app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsFragment.kt

```diff
@@ -73,7 +75,14 @@ class TransactionDetailsFragment : BaseViewBindingFragment<FragmentTransactionDe
                 }
                 is BaseStateViewModel.ViewAction.ShowError -> {
                     binding.refresh.isRefreshing = false
-                    //TODO: handle error here
+                    when(viewAction.error) {
+                        is Offline -> {
+                            snackbar(requireView(), R.string.error_no_internet)
+                        }
+                        else -> {
+                            //TODO: handle error
```

Comment:
Where should we address these todos?

Reply:
This TODO is addressed in origin/feature/793-error-and-empty-state-for-details